<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormularioSeguimientoEstudiante</class>
 <widget class="QDialog" name="FormularioSeguimientoEstudiante">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>557</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>510</width>
    <height>557</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>510</width>
    <height>557</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Formulario Estudiante</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">#frame{background-color: #f1f1f1;
border-top-left-radius: 10px;
border-top-right-radius: 10px; 
border-bottom-right-radius: 10px;
border-bottom-left-radius: 10px;}

</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QScrollArea" name="scrollArea">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>511</width>
        <height>461</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#scrollArea{background-color: #f1f1f1; 
border-bottom-right-radius: 10px;
border-bottom-left-radius: 10px;}



QLabel{
font-family: Roboto;
font-style: normal;
font-weight: 450;
font-size: 14px;
letter-spacing: 0.02em;
color: #6B7280;}


QLineEdit{
font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 14px;
letter-spacing: 0.02em;
background: #F3F4F6;
border: 2px solid #F3F4F6;
border-radius: 8px;
color:#9CA3AF;
padding:10px}

QLineEdit::focus{
background: #FFFFFF;
border: 2px solid #344647;
border-radius: 8px;
}



QDateEdit {
    font-family: Roboto;
    font-style: normal;
    font-weight: normal;
    font-size: 14px;
    letter-spacing: 0.02em;
    background: #F3F4F6;
    color: #9CA3AF;
    padding: 8px;
    border-radius: 8px;
}

QDateEdit::up-button {
    min-width: 25px;
    min-height: 12px;
    background-color: #F3F4F6; 
    border-radius: 6px;
}

QDateEdit::down-button {
    min-width: 25px;
    min-height: 12px;
    background-color: #F3F4F6; 
    border-radius: 6px;
}

QDateEdit::up-arrow {
    image: url(:/menu/contraerarriba.png);
    width: 11px;
    height: 11px;
}

QDateEdit::down-arrow {
    image: url(:/menu/contraerabajo.png);
    width: 11px;
    height: 11px;
}

QDateEdit:focus {
    background: #FFFFFF;
    border: 2px solid #344647;
    border-radius: 8px;
}

/* Estilo para el QCalendarWidget */
QCalendarWidget {
    font-family: Roboto;
    background: #F3F4F6;
    color: #344647;
    border: 2px solid #344647;
    border-radius: 8px;
}

QCalendarWidget QToolButton {
    background-color: #F3F4F6;
    color: #344647;
    font-size: 14px;
    border: none;
    margin: 5px;
    padding: 5px;
}

QCalendarWidget QToolButton:hover {
    background-color: #E2E3E4;
}

QCalendarWidget QMenu {
    background-color: #F3F4F6;
    color: #344647;
}

QCalendarWidget QSpinBox {
    background: #F3F4F6;
    color: #344647;
    font-size: 14px;
    border: 1px solid #E2E3E4;
    border-radius: 4px;
    margin: 2px;
}

QCalendarWidget QSpinBox::up-button, QCalendarWidget QSpinBox::down-button {
    subcontrol-origin: border;
    background: #E2E3E4;
    border: none;
}

QCalendarWidget QSpinBox::up-arrow, QCalendarWidget QSpinBox::down-arrow {
    width: 10px;
    height: 10px;
}

QCalendarWidget QAbstractItemView:enabled {
    font-size: 14px;
    color: #344647;
    background-color: #FFFFFF;
    selection-background-color: #689091;
    selection-color: #FFFFFF;
    outline: none;
}

QCalendarWidget QAbstractItemView:disabled {
    color: #9CA3AF;
}

QCalendarWidget QWidget#qt_calendar_navigationbar {
    background-color: #F3F4F6;
    border-bottom: 1px solid #E2E3E4;
}



/* QLabel para Mostrar Error */
QLabel#errorNombres, #errorApellidos, #errorIdentificacion, #errorFechaNacimiento, #errorCorreo, #errorTelefono, #errorDireccion{
    font-family: Roboto;
    font-style: normal;
    font-weight: bold;
    font-size: 14px;
    color: #D32F2F; 
}

QSpinBox {
    font-family: Roboto;
    font-style: normal;
    font-weight: normal;
    font-size: 14px;
    letter-spacing: 0.02em;
    background: #F3F4F6;
    color: #344647;
    border: 1px solid #E2E3E4;
    border-radius: 4px;
    padding: 8px;
    margin: 2px;
}

QSpinBox::up-button, QSpinBox::down-button {
    subcontrol-origin: border;
    background: #E2E3E4;
    border: none;
    width: 20px;
    height: 20px;
}

QSpinBox::up-arrow, QSpinBox::down-arrow {
    width: 10px;
    height: 10px;
}

QSpinBox::up-arrow {
    image: url(:/menu/contraerarriba.png);
}

QSpinBox::down-arrow {
    image: url(:/menu/contraerabajo.png);
}

QSpinBox::up-button:hover, QSpinBox::down-button:hover {
    background-color: #CCCCCC;
}


QPlainTextEdit{
font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 14px;
letter-spacing: 0.02em;
background: #F3F4F6;
border: 2px solid #F3F4F6;
border-radius: 8px;
color:#9CA3AF;
padding:10px}

QPlainTextEdit::focus{
background: #FFFFFF;
border: 2px solid #3a4f50;
border-radius: 8px;
}


</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scroll_registro_estudiante">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>494</width>
         <height>720</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>720</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">#scroll_registro_estudiante{
	background-color: rgb(255, 255, 255);
border-bottom-right-radius: 10px;
border-bottom-left-radius: 10px;
}
</string>
       </property>
       <widget class="QLabel" name="datospersonales">
        <property name="geometry">
         <rect>
          <x>7</x>
          <y>20</y>
          <width>161</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">#datospersonales {
    font-family: Roboto;
    font-style: normal;
    font-weight: bold;
    font-size: 17px;
    letter-spacing: 0.02em;
    color: #000000; 
    padding: 2px; 
    text-decoration: underline; /* Agrega subrayado */
}
</string>
        </property>
        <property name="text">
         <string>Datos Actividad</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_97">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>61</y>
          <width>161</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Fecha actividad</string>
        </property>
       </widget>
       <widget class="QDateEdit" name="fecha_actividad">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>90</y>
          <width>221</width>
          <height>40</height>
         </rect>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="displayFormat">
         <string>yyyy-MM-d</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="datosacademicos">
        <property name="geometry">
         <rect>
          <x>4</x>
          <y>460</y>
          <width>261</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">#datosacademicos {
    font-family: Roboto;
    font-style: normal;
    font-weight: bold;
    font-size: 17px;
    letter-spacing: 0.02em;
    color: #000000; 
    padding: 2px; 
    text-decoration: underline; /* Agrega subrayado */
}
</string>
        </property>
        <property name="text">
         <string>Evidencia de actividad realizada</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="horas_spinx">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>90</y>
          <width>211</width>
          <height>41</height>
         </rect>
        </property>
        <property name="maximum">
         <number>96</number>
        </property>
       </widget>
       <widget class="QLabel" name="label_98">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>60</y>
          <width>181</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Horas  actividad realizadas</string>
        </property>
       </widget>
       <widget class="QPlainTextEdit" name="actividades_plain">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>180</y>
          <width>471</width>
          <height>261</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_subir_fotos">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>500</y>
          <width>221</width>
          <height>35</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">
QPushButton {
    font-family: Roboto;
    font-style: normal;
    font-weight: normal;
    font-size: 15px;
    letter-spacing: 0.02em;
    color: white;
    padding: 8px 16px;
	background-color: #baba00;
    border: none; 
    border-radius: 8px; 
}

QPushButton:hover {
    background: #d3d300; 
    border: none;
    border-radius: 8px;
}


QPushButton:pressed {
    background: #ecec00;
    border: none;
    border-radius: 8px;
}</string>
        </property>
        <property name="text">
         <string>Subir fotos</string>
        </property>
        <property name="icon">
         <iconset resource="../image/source.qrc">
          <normaloff>:/menu/subir-imagen.png</normaloff>:/menu/subir-imagen.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>19</width>
          <height>19</height>
         </size>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_subir_archivo">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>500</y>
          <width>211</width>
          <height>35</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">
QPushButton {
    font-family: Roboto;
    font-style: normal;
    font-weight: normal;
    font-size: 15px;
    letter-spacing: 0.02em;
    color: white;
    padding: 8px 16px;
	background-color: #cf7c00;
    border: none; 
    border-radius: 8px; 
}

QPushButton:hover {
    background: #de8200; 
    border: none;
    border-radius: 8px;
}


QPushButton:pressed {
    background: #ff9500;
    border: none;
    border-radius: 8px;
}</string>
        </property>
        <property name="text">
         <string>Subir archivo</string>
        </property>
        <property name="icon">
         <iconset resource="../image/source.qrc">
          <normaloff>:/menu/archivo.png</normaloff>:/menu/archivo.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>19</width>
          <height>19</height>
         </size>
        </property>
       </widget>
       <widget class="QPlainTextEdit" name="observacion_plain">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>598</y>
          <width>471</width>
          <height>101</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="datosacademicos_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>571</y>
          <width>271</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">#datosacademicos {
    font-family: Roboto;
    font-style: normal;
    font-weight: bold;
    font-size: 17px;
    letter-spacing: 0.02em;
    color: #000000; 
    padding: 2px; 
    text-decoration: underline; /* Agrega subrayado */
}
</string>
        </property>
        <property name="text">
         <string>Observación ( opcional )</string>
        </property>
       </widget>
       <widget class="QLabel" name="lbl_nombre_archivo_seleccionado">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>540</y>
          <width>171</width>
          <height>16</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color:rgb(48, 165, 255);
font: italic 10pt &quot;Roboto&quot;;</string>
        </property>
        <property name="text">
         <string>ningun archivo seleccionado</string>
        </property>
       </widget>
       <widget class="QLabel" name="lbl_nombre_archivo_seleccionado_2">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>540</y>
          <width>171</width>
          <height>16</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color:rgb(48, 165, 255);
font: italic 10pt &quot;Roboto&quot;;</string>
        </property>
        <property name="text">
         <string>ningun archivo seleccionado</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_99">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>150</y>
          <width>221</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Lista de actividades realizados</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QFrame" name="frame_pie_pagina">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>500</y>
        <width>511</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#frame_pie_pagina{background-color: #f1f1f1; 
border-bottom-right-radius: 10px;
border-bottom-left-radius: 10px;}

/*  BOTON */

QPushButton {
    font-family: Roboto;
    font-style: normal;
    font-weight: normal;
    font-size: 15px;
    letter-spacing: 0.02em;
    color: white;
    padding: 8px 16px;
    background: #3f5758;
    border: none; 
    border-radius: 8px; 
}

QPushButton:hover {
    background: #4a6b6c; 
    border: none;
    border-radius: 8px;
}


QPushButton:pressed {
    background: #2e4546;
    border: none;
    border-radius: 8px;
}


/* Botón Cancelar */
QPushButton#cancelButton {
    font-family: Roboto;
    font-style: normal;
    font-weight: normal;
    font-size: 15px;
    letter-spacing: 0.02em;
    color: #3f5758;
    padding: 8px 16px;
    background: transparent;
    border: 2px solid #3f5758; 
    border-radius: 8px; 
}

QPushButton#cancelButton:hover {
    background: rgba(63, 87, 88, 0.1); /* Fondo ligeramente oscuro en hover */
    color: #2e4546; 
    border: 2px solid #2e4546;
}

QPushButton#cancelButton:pressed {
    background: rgba(63, 87, 88, 0.2); /* Fondo más oscuro en estado presionado */
    color: #1c2d2e; 
    border: 2px solid #1c2d2e;
}

</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="btn_agregar_actividad">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>10</y>
         <width>180</width>
         <height>40</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>110</width>
         <height>40</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Guardar actividad</string>
       </property>
       <property name="icon">
        <iconset resource="../image/source.qrc">
         <normaloff>:/menu/botonadd.png</normaloff>:/menu/botonadd.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>15</width>
         <height>21</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="cancelButton">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>10</y>
         <width>120</width>
         <height>40</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>40</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Cancelar</string>
       </property>
       <property name="icon">
        <iconset resource="../image/source.qrc">
         <normaloff>:/menu/no.png</normaloff>:/menu/no.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>15</width>
         <height>21</height>
        </size>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>511</width>
        <height>51</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">#frame_2{background: #3A4F50 ;border-top-left-radius: 10px;
border-top-right-radius: 10px;}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QToolButton" name="btn_closeedit">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>9</y>
         <width>25</width>
         <height>25</height>
        </rect>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="toolTip">
        <string>Cerrar</string>
       </property>
       <property name="styleSheet">
        <string notr="true">#btn_closeedit{
font-family: Roboto;
font-weight: normal;
font-size: 14px;
background-color: #d5d5d8;
color: #6B7280;
border-radius:11px
}



#btn_closeedit:hover{background:#ce4d3a ;
border-radius:11px; }

</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../image/source.qrc">
         <normaloff>:/menu/x.png</normaloff>:/menu/x.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>13</width>
         <height>13</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="lbl_titulo_ventana">
       <property name="geometry">
        <rect>
         <x>18</x>
         <y>14</y>
         <width>401</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 18px;
line-height: 23px;
letter-spacing: 0.02em;
color: #FFFFFF;}</string>
       </property>
       <property name="text">
        <string>Registrar Nuevo Actividad</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../image/source.qrc"/>
 </resources>
 <connections/>
</ui>
